extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -468,6 +468,7 @@ export class VisibilityManager {
       // Optionally, element-level state.
       let layoutBox;
       if (opt_element) {
+        state['elementId'] = opt_element.id;
```

Comment:
@zhouyx any concerns?

Reply:
I don't see any concern with this. 

There was proposal to include the element's tagName as well. Story team is adding story specific triggers. I'd propose that we use the same set of variable names in the future if we want to support elementId or similar. cc @gmajoulet @Enriqe

Reply:
SGTM. what is used in story right now? shall we just follow the same naming pattern?

Reply:
We currently don't have a custom variable for the elementId, only for tagName as seen here:

https://github.com/ampproject/amphtml/blob/0b9623cbc43521e28e395a58b9ab3613044cda14/extensions/amp-story/1.0/story-analytics.js#L25